lib/msal-common/src/request/AuthorizationUrlRequest.ts

```diff
@@ -18,6 +18,13 @@ export type AuthorizationUrlRequest = BaseAuthRequest & {
      */
     redirectUri: string;
 
+    /**
+     * The page that should be returned to after loginRedirect or acquireTokenRedirect. This should only be used
+     * if this is different from the redirectUri and will default to the page that initiates the request.
+     * When the navigateToLoginRequestUrl config option is set to false this parameter will be ignored.
+     */
+    redirectStartPage?: string;
```

Comment:
Because we are adding this parameter, and redirectURI is optional in msal-browser but required in msal-node, it may be better to have a wrapped request object that exists in `msal-browser` and use that for all calls:
```javascript
export type BrowserAuthorizationUrlRequest = AuthorizationUrlRequest & {
    redirectUri?: string;
    redirectStartPage?: string;
}
```